pkg/apiserver/server.go

```diff
@@ -183,42 +182,26 @@ func generateKotsadmID() error {
 		return nil
 	}
 	clusterID := clusters[0].ClusterID
-
-	// Write a Query to set/get an Event from the Store
-	isKotsadmIDGenerated, err := store.GetStore().IsKotsadmIDGenerated()
+	isKotsadmIDInDB, err := store.GetStore().IsKotsadmIDGenerated()
```

Comment:
why rename it? `isKotsadmIDGenerated` sounds better

Reply:
FWIW, is ID generated is kind of a weird name and I'm not sure it accurately reflects what this means. But renaming between the function and the variable seems wrong no matter what.

Reply:
@markpundsack how about `IsKotsadmIDMigrated`?

Reply:
let us go with IsKotsadmIDMigrated